### ISSUE TITLE
Multi-line comments don't terminate

### ISSUE THREAD

**Issue by chrisegner** [zanuka/dust-buster#11]
Syntax highlighting considers everything after a multi-line comments such as:

```
{!
        <div id="modal-container" class="modal-container-view hidden">{~n}
            {>"{modalView}" /}
        </div>{~n}
 !}       

```

to still be a comment.  This does not affect single-line comments.  

The salient detail seems to be the { in `</div>{~n}`.  Place a space before the { and it's fine.

**Comment by zanuka** [comment]
looking into this

**Comment by zanuka** [comment]
fixed in version/1.0.4
